generators/pkg/generator/generate.go

```diff
@@ -334,15 +400,20 @@ func GenerateTrigger(p *v1beta1.Pipeline) *trigger {
 							GitHub: &v1alpha1.GitHubInterceptor{
 								EventTypes: []string{"pull_request"},
 								SecretRef: &v1alpha1.SecretRef{
-									SecretKey:  "secretToken",
-									SecretName: "github-secret",
+									SecretKey:  g.Spec.SecretKey,
+									SecretName: g.Spec.SecretName,
 								},
 							},
 						},
+						{
+							CEL: &v1alpha1.CELInterceptor{
+								Filter: "body.head.ref == " + g.Spec.Branch,
```

Comment:
```suggestion
								Filter: "body.base.ref == " + g.Spec.Branch,
```

I think we want base here, since this is the ref you want to merge into. e.g. for this PR, base == master and head == generators-pr.